packages/context/README.md

```diff
@@ -206,101 +206,12 @@ absolute URL.
-
-##### `documentChanges.subscribe('{objectId}/{instanceId}', (data: DocumentChange[]]) => {})`
-
-Subscribe to the specified object instance document changes.
-
-```javascript
-const { documentChanges } = useSignalRConnection();
-
-documentChanges.subscribe('myObjectId/myInstanceId', (data) => {
-    console.log(data);
-});
-```
-
-The data provided to the callback will be an array of `DocumentChange` which contains the
-following data:
-
--   `objectId`
-    -   Object describing the instance associated with the updated document.
--   `instanceId`
-    -   Instance that the updated document is associated with.
--   `documentId`
-    -   Document that was updated.
--   `type`
-    -   The type of update. Possible values are `BlobCreated`, `BlobDeleted`, and `BlobMetadataUpdated`.
-
-##### `documentChanges.unsubscribe('{objectId}/{instanceId}', (data: DocumentChange[]) => {})`
-
-Unsubscribe to the specified object instance document changes.
-
-Callback function is optional.
-If callback function is defined, you must pass the exact same Function instance as was previously passed to `documentChanges.subscribe`.
-Passing a different instance (even if the function body is the same) will not remove the subscription.
-
-```javascript
-const { documentChanges } = useSignalRConnection();
-
-const callback = (data: DocumentChange[]) => {
-    console.log(data);
-};
-
-documentChanges.subscribe('myObjectId/myInstanceId', callback);
-
-documentChanges.unsubscribe('myObjectId/myInstanceId', callback);
-```
-
-##### `instanceChanges.subscribe('{objectId}', (instanceIds: InstanceChange[]) => {})`
-
-Subscribe to the specified object instance changes.
-
-```javascript
-const { instanceChanges } = useSignalRConnection();
-
-instanceChanges.subscribe('myObjectId', (instanceIds) => {
-    console.log(instanceIds);
-});
-```
-
-The data provided to the callback will be an array of instance IDs that were updated.
-
-##### `instanceChanges.unsubscribe('{objectId}', (instanceIds: InstanceChange[]) => {})`
-
-Unsubscribe to the specified object instance changes.
-
-Callback function is optional.
-If callback function is defined, you must pass the exact same Function instance as was previously passed to `instanceChanges.subscribe`.
-Passing a different instance (even if the function body is the same) will not remove the subscription.
-
-```javascript
-const { instanceChanges } = useSignalRConnection();
-
-const callback = (instanceIds: InstanceChange[]) => {
-    console.log(instanceIds);
-};
-
-instanceChanges.subscribe('myObjectId', callback);
-
-instanceChanges.unsubscribe('myObjectId', callback);
-```
-
-### Notification
+### Notifications
 
 -   [useNofitication](#usenotification)
     -   [documentChanges](#documentchangessubscribeobjectidinstanceid-data-documentchange)
     -   [instanceChanges](#instancechangessubscribeobjectid-instanceids-instancechange)
+-   [~~useSignalRConnection~~](#usesignalrconnection) (_Deprecated_)
```

Comment:
@actong We might not want to use the term "deprecated" for it -- it implies that it still functions as before but can become non-functional later so its use is not encouraged. From the description below it seems like it doesn't really work as intended any more, it just won't be breaking any code (which we anyways have to ensure).

I think we normally use "Obselete" for such cases. Please see this for reference: https://www.baeldung.com/cs/deprecated-vs-obsolete